log/formatter/general_test.go

```diff
@@ -0,0 +1,209 @@
+package formatter
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/suite"
+
+	configmock "github.com/goravel/framework/contracts/config/mocks"
+)
+
+type GeneralFormatterTestSuite struct {
+	suite.Suite
+	mockConfig  *configmock.Config
+	entry       *logrus.Entry
+	stackTraces any
+}
+
+func TestGeneralFormatterTestSuite(t *testing.T) {
+	suite.Run(t, new(GeneralFormatterTestSuite))
+}
+
+func (s *GeneralFormatterTestSuite) SetupTest() {
+	s.mockConfig = &configmock.Config{}
+	s.entry = &logrus.Entry{
+		Level:   logrus.InfoLevel,
+		Message: "Test Message",
+	}
+
+	s.mockConfig.On("GetString", "app.timezone").Return("UTC")
+	s.mockConfig.On("GetString", "app.env").Return("test")
+}
+
+func (s *GeneralFormatterTestSuite) TestGeneral_Format() {
+	general := NewGeneral(s.mockConfig)
+	formatLog, err := general.Format(s.entry)
+	s.Nil(err)
+	s.NotNil(formatLog)
+}
+
+func (s *GeneralFormatterTestSuite) TestFormatData() {
+	tests := []struct {
+		name   string
+		setup  func()
+		assert func()
+	}{
+		{
+			name: "Data is empty",
+			setup: func() {
+				s.entry.Data = logrus.Fields{}
+			},
+			assert: func() {
+				formattedData, err := formatData(s.entry.Data)
+				s.Nil(err)
+				s.Empty(formattedData)
```

Comment:
Q: Can we judge a specific content?

Reply:
`formatData` will return formatted String so if we want to judge a specific content, ig we can use check that in formattedData